security/openacademy_security.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="group_openacademy_manager" model="res.groups">
+        <field name="name">OpenAcademy / Manager</field>
+        <field name="comment">Security group for managers.</field>
+    </record>
+
+    <record id="group_openacademy_user" model="res.groups">
+        <field name="name">OpenAcademy / User</field>
+        <field name="comment">Security group for users.</field>
+    </record>
+
+    <record id="restricts_responsible" model="ir.rule">
+        <field name="name">Restricts write and unlink accesses.</field>
+        <field name="model_id" ref="openacademy.course"/>
```

Comment:
I'm pretty sure this doesn't work.

Reply:
I get a parse error, any ideas?

Reply:
`ref` is for external IDs. I think you meant:

`<field name="model_id" ref="model_openacademy_session"/>`

Reply:
Problem solved.